jetty-util/src/main/java/org/eclipse/jetty/util/log/JavaUtilLog.java

```diff
@@ -65,8 +65,8 @@
 {
     private static final String THIS_CLASS = JavaUtilLog.class.getName();
     private static final boolean __source =
-        Boolean.parseBoolean(Log.__props.getProperty("org.eclipse.jetty.util.log.SOURCE",
-            Log.__props.getProperty("org.eclipse.jetty.util.log.javautil.SOURCE", "true")));
+        Boolean.parseBoolean(Log.getProperty("org.eclipse.jetty.util.log.SOURCE",
+            Log.getProperty("org.eclipse.jetty.util.log.javautil.SOURCE", "true")));
 
     private static boolean _initialized = false;
```

Comment:
Why this `static` has just 1 `_` while the same in `Log` has 2 `__`?

Reply:
It was `__<constant>` (aka `static final`) and `_<static>` (aka just `static`)

Reply:
I'll fix it anyway.